docker-compose.yml

```diff
@@ -25,7 +25,7 @@ services:
             - "3000:3000"
 
     test:
-        image: mcr.microsoft.com/playwright:v1.39.0-jammy
+        image: mcr.microsoft.com/playwright:latest
```

Comment:
Why switch to `latest`?

Reply:
One reason why is that otherwise we just have to manually bump the version ever so often. The [first](https://github.com/dhis2-sre/im-web-client/actions/runs/6978199671/job/18989198135) test run failed and the a bump was necessary.

Another reason is that our tests should be able to run on the latest image. If they fail we can pin them to a specific version or update the failing test.